std-bits/base/src/main/java/org/enso/base/XML_Utils.java

```diff
@@ -20,9 +19,11 @@ public class XML_Utils {
    *
    * @param element the element to convert to a string
    * @return the string representation of the element
-   * @throws TransformerException
+   * @throws ClassNotFoundException if the DOM implementation class cannot be found.
+   * @throws IllegalAccessException if the DOM implementation class cannot be accessed.
+   * @throws InstantiationException if the DOM implementation class cannot be instantiated.
```

Comment:
I have a hard time understanding what is the difference between being able to _access_ the class from _instantiate_ it.

Do we need the distinction? Couldn't we throw the same kind of exception for these cases?

Is this exception even expected at all? Isn't it some kind of internal error if it occurs?